backend/controllers/LocationsController.ts

```diff
@@ -0,0 +1,45 @@
+import { Response, Request } from "express";
+import Location from "../models/Locations";
+
+const LocationController = {
+  async getLocationId(req: Request, res: Response) {

```

Comment:
rename to getLocationById
(we are not getting not an id, but the location by it)